user_docs/en/userGuide.md

```diff
@@ -280,6 +280,74 @@ To open NVDA's menu, press `NVDA+n`.
+1. Use the `arrow keys` or press the first letter of an add-on name to move around the list.
+1. Press `tab` once to move to a description of the currently selected add-on.
+1. Use the [reading keys](#ReadingText) or `arrow keys` to read the full description.
+1. Press `tab` to the "Actions" button, which can be used to install the add-on, among other actions.
+1. Press `tab` to "Other Details", which lists details such as the publisher, version and homepage.
+1. To return to the list of add-ons, press `alt+a`, or `shift+tab` back to the "Available add-ons" list.
+
+#### Searching for add-ons {#SearchingForAddons}
+As well as browsing all available add-ons, it is possible to filter the add-ons shown.
+To search, press `alt+s` to jump to the "search" field and type the text to search for.
+The search field can contain information including the add-on ID, display name, publisher, author or description.
+The list changes while typing the search terms.
+Once done, press `tab` to go to the filtered list of add-ons and browse the results.
+
+#### Installing add-ons {#InstallingAddons}
+
+To install an add-on:
+1. With the focus on an add-on you would like to install, press `enter`.
+1. The actions menu opens with a list of actions including: "Install", "Home page", "License", "Source code", "Community reviews".
+1. To install the add-on, press `i` or `down arrow` to "install" and press `enter`.
+1. The focus returns to the add-on in the list and NVDA will read the details about the add-on.
+1. The "Status" information reported by NVDA changes from "Available" to "Downloading".
+1. Once the add-on has finished downloading, it will change to "Downloaded. Pending install".
+1. Repeat with any other add-ons you would like to install at the same time.
+1. Once finished, press `tab` until the focus is on the "Close" button, then press `enter`.
+1. A dialog appears advising that changes were made, and you must restart NVDA.
+1. Press `enter` to restart NVDA.
+
+#### Managing installed add-ons {#ManagingInstalledAddons}
+Press `control+tab` to move between the tabs or sections of the Add-on Store.
+The tabs include: "Installed add-ons", "Updatable add-ons", "Available add-ons" and "Installed incompatible add-ons".
+Each of the tabs are set out similar to each other, as a list of add-ons, a panel for more details on the selected add-on, and the ability to perform actions for the add-on.
+The actions menu of installed add-ons includes "Disable", and "Remove" rather than "Install".
+Disabling an add-on stops NVDA from loading it, but leaves it installed.
+To re-enable a disabled add-on, activate "Enable" from the actions menu.
+After enabling, disabling, or removing add-ons, NVDA will prompt to restart when closing the add-on store.
+Note that in this dialog `escape` works the same as the close button.
+Any changes you had made in the dialog have already happened, but they don't take effect until NVDA restarts.

```

Comment:
I find this sentence a little confusing - even though I was the one who suggested something like this in the first place.

```suggestion
Any changes you had made to add-ons such as disabling or removing them cannot be undone until NVDA restarts.
```

Reply:
Perhaps we could just simplify it to:
```suggestion
NVDA must be restarted before changes such as removing or disabling add-ons take effect.

```

Reply:
I think that should be covered by the previous sentence?

Reply:
After enabling, disabling, or removing add-ons, NVDA will prompt to restart when closing the add-on store.
Note that in this dialog `escape` works the same as the close button.
NVDA must be restarted before changes such as removing or disabling add-ons take effect.

Ok perhaps:

```suggestion
Any changes such as removing or disabling add-ons take effect after restarting

```